README.md

```diff
@@ -18,33 +18,20 @@
 - <strong>Performance</strong>: Keep yourself calm. We use strict budgets to ensure our libs are always light and fast.
 - <strong>Stability</strong>: Generate your pages ahead by using a Jamstack ready toolkit.
 - <strong>SEO/Analytics</strong>: Connect to your favorite analytics provider.
-- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce in your favorite React framework
-  (Gatsby, Next.JS, Remix etc)
-- <strong>Speed</strong>: Customize one of our starters to your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.
+- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce using Next.JS, a powerful React framework.
+- <strong>Speed</strong>: Customize our starter to fit your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.
 
 ## Getting started
 
-[Check out the docs](https://faststore.dev/quickstart) to quickstart from one of our official starters or create your own.
+[Check out the docs](https://faststore.dev/) to quickstart from our official starter and start creating.
 
-## Official Starters
+## Official Starter
 
-- [Gatsby](https://github.com/vtex-sites/gatsby.store): Basic starter with a generic store.
-- [NextJS](https://github.com/vtex-sites/nextjs.store): Basic starter with a generic store.
+- [NextJS](https://github.com/vtex/faststore): Basic starter with a generic store.
 
 ## Learn
 
-Check out our [Tutorials](https://faststore.dev/tutorials), [How-to Guides](https://faststore.dev/how-to-guides), [API References](https://faststore.dev/reference) and [Conceptual guides](https://faststore.dev/conceptual-guides) to deep dive in the world of building FastStores.
-
-## Release Notes
-
-Follow our monthly release notes at: https://faststore.dev/releases
-
-## Who's using FastStore?
-
-These are the clients running in production with FastStore:
-
-- [Marin Brasil](https://www.marinbrasil.com.br/)
-- [Carrefour](https://mercado.carrefour.com.br/)
+Check out our [Overview](https://www.faststore.dev/docs/getting-started-overview), [How-to Guides](https://www.faststore.dev/docs/building-sections/component-customization-overview), [API References](https://faststore-site-git-api-extension-development-faststore.vercel.app/docs/api-extensions) and [Glossary](https://www.faststore.dev/docs/getting-started-glossary) to deep dive in the world of building FastStores.
 
 ## Contributing
```

Comment:
What about a link to the [CONTRIBUTING.MD](https://github.com/vtex/faststore/blob/docs/update-readme/CONTRIBUTING.MD) as well? For example:

> This is still a work in progress. However, if you are also adventurous, you can read the code and have fun! Remember to check out our  [CONTRIBUTING.MD](https://github.com/vtex/faststore/blob/docs/update-readme/CONTRIBUTING.MD).

Reply:
sounds good, but I think this file is out of date :/